src/sentry/runner/commands/run.py

```diff
@@ -677,6 +678,11 @@ def handler(signum, frame):
 def metrics_billing_consumer(**options):
     from sentry.ingest.billing_metrics_consumer import get_metrics_billing_consumer
 
+    if "max_batch_time" in options:
+        logger = logging.getLogger("billing-metrics-consumer")
+        logger.info("`max_batch_time` not supported, falling back to default once per second.")
```

Comment:
nit: Any reason this is not a warning?

Reply:
Not really. I added this as a warning initially, and not strongly opinionated so I'm open to making it a warning again.

My reasoning is basically how a message should be interpreted. In my opinion, logs with levels from warning (inclusive) upwards should be something to look at and address them sooner than later (priority determined by the log level). That message is going to be logged on _every_ run, which would generate too much noise yet there's no need to take any action; however, I consider it more important than `debug` and worth seeing in the terminal when running manually, so I chose `info` between both.

Reply:
You could also argue that making the log `info` based on that reasoning is kind of cheating, and that's a good point 😄.